model/src/main/kotlin/config/OrtConfiguration.kt

```diff
@@ -109,13 +109,14 @@ data class OrtConfiguration(
 
     /**
      * The package curation providers to use. Defaults to providers for the default [ORT_PACKAGE_CURATIONS_FILENAME] and
-     * [ORT_PACKAGE_CURATIONS_DIRNAME] configuration locations. The order of this list defines the priority of the
-     * providers: Providers that appear earlier in the list can overwrite curations for the same package from providers
-     * that appear later in the list.
+     * [ORT_PACKAGE_CURATIONS_DIRNAME] configuration locations as well as the provider for Spring packages. The order of
+     * this list defines the priority of the providers: Providers that appear earlier in the list can overwrite
+     * curations for the same package from providers that appear later in the list.
      */
     val packageCurationProviders: List<ProviderPluginConfiguration> = listOf(
         ProviderPluginConfiguration(type = "DefaultDir"),
-        ProviderPluginConfiguration(type = "DefaultFile")
+        ProviderPluginConfiguration(type = "DefaultFile"),
+        ProviderPluginConfiguration(type = "Spring")
```

Comment:
Why should this be enabled by default? 

This changes the standard workflow to on the fly generated curations, which might break some workflows such as replacing curations in an ORT result and makes it harder to figure out where some curation came from. So, I wonder if it's better to leave this disabled?